testing/tests/DevExpress.ui.widgets/treeViewParts/optionChanged.js

```diff
@@ -155,6 +155,36 @@ QUnit.test("searchValue from value to empty - update selection", function(assert
     assert.equal($items.length, 6, "6 items were rendered after filtration");
 });
 
+QUnit.test("Should recalculate selection after 'searchValue' changing", function(assert) {
+    var treeView = initTree({
+            searchValue: "b",
+            showCheckBoxesMode: "normal",
+            items: [{
+                id: "all",
+                text: "all",
+                items: [
+                    { id: "b", text: "b" },
+                    { id: "c", text: "c" }
+                ]
+            }]
+        }).dxTreeView("instance"),
+        $treeView = $(treeView.$element());
+
+    var checkSelection = function(isFirstItemSelected, isSecondItemSelected) {
```

Comment:
IInline this method. When all items will be rendered, it will be not necessary to find them before each asserrt